trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/generate_propeller_wake_distribution.py

```diff
@@ -12,10 +13,9 @@
 import numpy as np
 from SUAVE.Core import Data 
 from SUAVE.Methods.Aerodynamics.Common.Fidelity_Zero.Lift.compute_wake_contraction_matrix import compute_wake_contraction_matrix
-from SUAVE.Methods.Geometry.Three_Dimensional import  orientation_product, orientation_transpose
 
 ## @ingroup Methods-Aerodynamics-Common-Fidelity_Zero-Lift   
-def generate_propeller_wake_distribution(prop,m,VD,init_timestep_offset, time, number_of_wake_timesteps ): 
+def generate_propeller_wake_distribution(props,m,VD,init_timestep_offset, time, number_of_wake_timesteps,conditions ): 
```

Comment:
Thinking about it, we should add rotors too, for like a Joby cofiguration

Reply:
This nomenclature will change with Racheals PR.